src/plugin/analytics/Resources/modules/administration/dashboard/components/log.jsx

```diff
@@ -30,20 +35,15 @@ const DashboardLog = () =>
           label: trans('details'),
           displayed: true
         }, {
-          name: 'doer.name',
-          type: 'string',
-          label: trans('doer'),
-          displayed: true
-        }, {
-          name: 'target.name',
-          type: 'string',
+          name: 'target',
+          type: 'user',
           label: trans('target'),
-          displayed: true
+          displayed: false
         }, {
           name: 'event',
-          type: 'string',
+          type: 'translation',
           label: trans('event'),
-          displayed: true
+          displayed: false
```

Comment:
`options: {domain: 'security'}` to follow changes requested in translations.

Reply:
Done, thx :)